blocks/bricks/sequence_generators.py

```diff
@@ -582,6 +582,19 @@ class AbstractEmitter(Brick):
 
     :class:`SoftmaxEmitter` : for integer outputs
 
+    Notes
+    -----
+    An important detail about the emitter cost is that it will be
+    evaluated with inputs of different dimensions so it has to be
+    flexible enough to handle this. The two ways in which it can be
+    applied are:
+
+        1. In the cost_matrix function of :class:`BaseSequenceGenerator`
```

Comment:
Can you use `:meth:\`BaseSequenceGenerator\.cost_matrix`` since it will be rendered and create links to these methods.